packages/manager/apps/sign-up/src/form/form.controller.js

```diff
@@ -65,6 +67,15 @@ export default class SignUpFormAppCtrl {
     this.saveError = null;
     this.loading.init = true;
 
+    /**
+     * temporary fixed to avoid inherit another language for US customers
+     * TODO: It's better to have a feature to allow customers change the language as we have across Manager
+     */
+    const { lang } = this.$location.search();
+    if (this.coreConfig.isRegion('US') && lang !== US_DEFAULT_LANGUAGE) {
+      this.$location.search('lang', US_DEFAULT_LANGUAGE);
+    }
```

Comment:
Lets not handle the default language setting in this way. Its necessary to have `lang` parameter in the URL and lets findout how Sadia was not able to get that URL and fix the URL.